recipes-core/os-release/os-release.bbappend

```diff
@@ -0,0 +1,7 @@
+PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
Now we no longer need this line since the os-release package isn't machine dependent :)

Reply:
Right!  I removed that and also DEFAULT_HOSTNAME from the list of tags.